### PR TITLE
[JENKINS-39992] Fix has() on Pager

### PR DESCRIPTION
**Pull request by imeredith**
# Description

Removed the preinstall step in package.json because it breaks 'npm install ../blueocean-core-js'

See [JENKINS-39992](https://issues.jenkins-ci.org/browse/JENKINS-39992).

# Submitter checklist
- [ ] Link to JIRA ticket in description, if appropriate.
- [ ] Change is code complete and matches issue description
- [ ] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [ ] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.
- [ ] Ran Acceptance Test Harness against PR changes.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

@reviewbybees

### CODE REVIEW

File: blueocean-core-js/package.json [hunk 1]
```diff
@@ -6,8 +6,7 @@
   "scripts": {
     "gulp": "gulp",
     "test": "gulp test",
-    "prepublish": "gulp",
-    "preinstall": "gulp test"
+    "prepublish": "gulp"
```
Comment (michaelneale) [inline blueocean-core-js/package.json]: how come not running test? no need to explicitly call it?

Comment (imeredith) [inline blueocean-core-js/package.json]: Because it breaks npm install ../blueocean-core-js usecase

**Comment by michaelneale** [comment]
thanks @imeredith could you include https://github.com/jenkinsci/blueocean-acceptance-test/pull/72 - or a rewrite of it to validate this fix?

**Comment by imeredith** [comment]
This is covered by https://github.com/jenkinsci/blueocean-acceptance-test/pull/70 - need a bee then can merge.

**Comment by michaelneale** [comment]
:bee: don't forget to update core-js etc.

**Pull request merged**